docs/blog/2019-04-02-behind-the-scenes-what-makes-gatsby-great/index.md

```diff
@@ -16,7 +16,7 @@ Gatsby is **great** from a multititude of perspectives. Our community is **great
 As such--this post focuses on just a single element of what makes Gatsby great: performance. To prime the discussion, let's consider [this post on the `webdev` subreddit on Reddit](https://www.reddit.com/r/webdev/comments/9z5dsr/how_does_reactjs_have_such_a_fast_website/?st=jtqbllhm&sh=60148ea7).
 
 <Pullquote citation="reddit/r/webdev">
-Genuine question, every page is loaded immediatley [sic] on click. Seriously never seen such a quick website before. Any insight as to how they're able to achieve this?
+Genuine question, every page is loaded immediately [sic] on click. Seriously never seen such a quick website before. Any insight as to how they're able to achieve this?
```

Comment:
We should remove the [sic] if the typo was fixed–that was intentionally kept-in as a quote.

Reply:
what does [sic] mean? oh let me revert that

Reply:
When you quote someone/something but the quote has a typo (as in the example above) the `[sic]` signals that not you but the original author made that typo. You normally don't want to edit quotes.